server/database.py

```diff
@@ -47,10 +49,11 @@ def status(self, value):
         self.status_level = User.STATUS_LIST.index(value)
 
     @classmethod
-    def get(cls, id, *args, **kwargs):
+    def get(cls, id, name="Unknown", *args, **kwargs):
```

Comment:
Having `name` on users is fine, would you like this to be user-set or should it be their main's character name? If it should be their main's character name, this code should use `user.name = Character.get(id).name`.

Reply:
yes but do we want to call Character.get from inside the constructor?